apps/admin-portal/src/components/identity-providers/wizards/outbound-provisioning-connector-create-wizard.tsx

```diff
@@ -0,0 +1,427 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import * as React from "react";
```

Comment:
```suggestion

```

Reply:
@inthirakumaaran we should also add `React` import to the existing import as well.